### ISSUE TITLE
cell cycle phenotypes

### ISSUE THREAD

**Issue by ValWood** [pombase/fypo#3056]
I know we discussed this before, but   I think we need to look again.

Re https://helpdesk.ebi.ac.uk/Ticket/Display.html?id=200002&results=95b3d35f13245b8cf45402b9ecc492a3

I was thinking these numbers were quite low because in
https://www.ncbi.nlm.nih.gov/pubmed/23697806
we identified 513 genes that were required for cell cycle progression  (elongated, vegetative). 
So I would add 
FYPO:0001490 - inviable elongated vegetative cell  
and
FYPO:0001492 - viable elongated vegetative cell
to this list.

The general assumption is  that all of these phenotypes are due to some sort of cell cycle transition or cytokinesis problem....Although I’m not sure about germinated cells that end up with an elongated germ tube

http://pombase2.aska.gen.nz/term/FYPO%3A0001490
http://pombase2.aska.gen.nz/term/FYPO%3A0001492

We need to check and see what we need to do to get them to be annotated transitively to "abnormal cell cycle transition".

Will discuss with ~@haylesjv~   @JackyVH next week.

**Comment by ValWood** [comment]
Need to find previous discussions.

**Comment by ValWood** [comment]
So, of the descendants of 
FYPO:0001492 - viable elongated vegetative cell

These would normally be a G2/M delay
viable after spore germination with elongated germ tube
but would need additional fifferentia, I'll return to this one
--

viable elongated vegetative cell, with progressive elongation
should be related to "delayed mitotic G2/M transition"

FYPO:0003481 - viable elongated vegetative cell, elongated upon mitotic entry
should be related to "delayed mitotic G2/M transition"

both of these are phenotypes used to indicate a G2/M transition delay

**Comment by ValWood** [comment]
we have a whole heap of things annotated to this term directly  (336)
http://preview.pombase.org/term_single_allele_genotype_genes/FYPO%3A0001492
mainly from deletion project.

Usually when we use this term, we are talking about a vegetative G2/M delay.

We should add a comment to this term
If your cell is elongated  and divides normally, you should use the descendant term
FYPO:0003481 - viable elongated vegetative cell, elongated upon mitotic entry
(I don't think we have been doing this consistently).

Now I need to think of a way to correct the existing.

**Comment by ValWood** [comment]
inviable after spore germination, multiple cell divisions, cell cycle arrest in mitotic interphase, elongated cells

this term should have a  G2/M transition relationship

**Comment by mah11** [comment]
> we have a whole heap of things annotated to [FYPO:0001492] directly 

I suspect if I've used FYPO:0001492 directly, it would be for experiments where I wouldn't be able to tell whether the cell divides normally.

(note to self: request for FYPO:0003481, and distinction from FYPO:0001492, in #1533)

**Comment by mah11** [comment]
> inviable after spore germination, multiple cell divisions, cell cycle arrest in mitotic interphase, elongated cells
> 
> this term should have a G2/M transition relationship

It can't - "G2/M transition" is more specific than "mitotic interphase".

**Comment by ValWood** [comment]
I think even these can have the G2/M transition delay:


- [ ] inviable elongated mononucleate aseptate vegetative cell with cell cycle arrest in mitotic M phase

- [ ] inviable elongated mononucleate aseptate vegetative cell with cell cycle arrest in mitotic metaphase


but would need @JackyVH  to confirm.
The cells are elongated so there is a transition delay. Then they are inviable  because they got into M-phase with unresolved problems, or experienced further problems that were not corrected.

**Comment by mah11** [comment]
While it's technically true that if a cell is arrested at any point in M phase, the G2/M transition will take place later than normal (if at all), I can't help thinking it would be not only tidier, but easier to understand if the G2/M transition delay term continued to mean that things stall specifically at the point when the transition would normally take place. In other words, right now the G2/M transition delay term means that cells get as far as the G2/M transition point but then don't go through the transition.

**Comment by ValWood** [comment]
> I suspect if I've used FYPO:0001492 directly, it would be for experiments where I wouldn't be able to tell whether the cell divides normally.

Yes the only reason to use this term is if you cant see whats happening with nucleus or septa.
(I'm using 'divides normally' loosely here.

In vegetative growth cells are elongated for 2 reasons:
 G2/M delay
problems with cytokinesis (branched, multinucleate, multiseptate)
Cells can be longer when they diploidize, but that is a bit different... they are also hugely swollen (so I'm not even sure if they are elongated)

**Comment by ValWood** [comment]
>While it's technically true that if a cell is arrested at any point in M phase, the G2/M transition will take place later than normal (if at all),

That isn't technically true because G2/M precedes M phase.
The issue here is that these cells have more than one problem at the same time, and one problem is a G2/M transition delay


> I can't help thinking it would be not only tidier, but easier to understand if the G2/M transition delay term continued to mean that things stall specifically at the point when the transition would normally take place. In other words, right now the G2/M transition delay term means that cells get as far as the G2/M transition point but then don't go through the transition.

Hmm, I have never used it like this. To most cell cycle people "G2/M transition delay"
 applies to 
a) things which halt at the transition (indicated by elongated cells, because mitosis is blocked, but cells keep growing because they are still in G2)
 b) all things which get through the transition, but are elongated when they do so.
This phenotype (elongated at division) is used to indicate a G2/M transition delay. The cells which are elongated at division time indicate a G2/transition delay  occurred (even if the delay is transient, i.e checkpoint delay to resolve problems). But the phenotype use to indicate a G2 delay   is  elongated cells at division time.

**Comment by mah11** [comment]
>> if a cell is arrested at any point in M phase, the G2/M transition will take place later than normal
> 
> That isn't technically true because G2/M precedes M phase.

oops, yep, ignore that part.

The rest, about how G2/M transition delay is defined, still holds. If things go through the transition, but do so later than normal, that fits the definition. I admit I'm not sure I follow what the difference between a (halt at transition) and b (are longer when they go through the transition) is. Is it that cells in (a) attempt the transition at normal size, fail transition, and then get longer than normal?

At present we have not defined these terms as having G2/M transition delay as a part:

> inviable elongated mononucleate aseptate vegetative cell with cell cycle arrest in mitotic M phase
> inviable elongated mononucleate aseptate vegetative cell with cell cycle arrest in mitotic metaphase

... but if they necessarily do, we can add the has_part link in the ontology (not is_a).

**Comment by ValWood** [comment]
Any elongated vegetative cell is  because some G2/M transition checkpoint is active, or other signals to enter mitosis are not satisfied (so cdc2 does not become activated, or the balance of activated vs. inactivated is not achieved.)

This could be either
DNA replication checkpoint /Damage checkpoint signalling
some stress and nutrient related signalling or the cell polarity pathway .

 The consequence of all of these "interphase blocks) is that cells do not divide at the "normal size". They are kept longer in interphase and while they are in interphase they keep growing. This manifest in a G2/M transition delay.


Although I have thought of more problems. We define normal size as the size in standard lab nutrient conditions. Cells delay mitotic entry related to this size if nutrients are plentiful (they divide at a longer size), or smaller if nutrients are limited, so this depends on our interpretation of "normal". That might be OK..... 

Basically the elongated size is what you see whenever the transition is delayed related to "normal". So maybe it would work as a has_output?  

Maybe we should concentrate first on
FYPO:0003481 - viable elongated vegetative cell, elongated upon mitotic entry
 and how that should relate to  to "delayed mitotic G2/M transition"

FYPO:0003481 - viable elongated vegetative cell, elongated upon mitotic entry
when it occurs is always caused by  "delayed mitotic G2/M transition"
what would that relation be?

Then we can see which annotations  on the parent can move down to this term.

Then we can assess the other terms after this.

**Comment by JackyVH** [comment]
inviable elongated mononucleate aseptate vegetative cell with cell cycle arrest in mitotic M phase
inviable elongated mononucleate aseptate vegetative cell with cell cycle arrest in mitotic metaphase

Both of these  could be the result of checkpoint activation , during which the cells will elongate, due say the damage in S phase . In the absence of repair or what ever, the checkpoint may be inactivated/bypassed and the cells enter mitosis and block because of the defect during S phase. So there is a G2M delay causing cell elongation and a block in M because of a defect during S phase

**Comment by ValWood** [comment]
so Jacky is saying pretty much the same thing here


> Is there another way to differentiate the cytokinesis block elongated from G2/M?  would the cytokinesis ones always have miltipe nuclei and/or multiple septa?

Exactly. The delay at G2/M will be  aseptate elongated mononucleate cells. It is possible that some cells will leak through and septate  particularly ts mutants but this is usually at late time points and the major phenotype will be the aseptate elongated mononuclear cell, where as the cytokinesis defect/delay will have at least two nuclei

so its as follows I think

inviable elongated aseptate cell. This would need DAPI or some thing that stains the nucleus to confirm it it mononuceate and therefore a G2 block or multinucleated and therefore can go through mitosis normally  but has a septation defect and cannot form a septum inviable elongated aseptate mono nucleate cell cell  - G2/M delay
inviable elongated aseptate multinucleate cell -septation defect ( cannot form a septum)
inviable branched septated cell  -  likely to be a septation defect (growth reinitiated ) needs DAPI staining to confirm multinucleate
inviable branched septated multinucleate cell-  septation defect (growth reinitiated )
viable elongated septated cell normal septation index- G2/M delay
viable elongated septated cell high septation index - septation defect

>
> Does that make sense?

Yes it does. Thinking about it a bit more I think the number of nuclei is the key thing. I hope this makes sense

Jacky

so if the cells are elongated at division time, and its clearly a "single division" it's a G2 delay...

**Comment by mah11** [comment]
> We define normal size as the size in standard lab nutrient conditions.

No, FYPO defines normal size as the size of wild type 972h- (or 975 h+ or 968 h90) cells under the conditions used in the experiment. At present this isn't as explicit as perhaps it could be, but it would merely be a bit of a slog, not conceptually or technically difficult, to add to the text definitions to spell it out.

**Comment by JackyVH** [comment]
Thinking about this more I think that to really say it is a G2M block/delay you need FACS to show that it is post replication e.g. cdc10 does not activate the checkpoint but blocks as an elongated mononucleate aseptate cell

**Comment by JackyVH** [comment]
PS I think cdc10 is the only gene that does this. Post cdc10 if the checkpoint is not activated cells cut

**Comment by ValWood** [comment]
> Thinking about this more I think that to really say it is a G2M block/delay you need FACS to show that it is post replication e.g. cdc10 does not activate the checkpoint but blocks as an elongated mononucleate aseptate cell

Ah I see, so this would  equivalent be an S-phase block with a cell that keeps growing?

**Comment by JackyVH** [comment]
well sort of but cells blocked in S  will activate the checkpoint because cdc2-cyclin is present and if not inactivated will allow cells to enter mitosis (eventually).  The cdc10 block is at the G/S before cdc2-cyclin starts to accumulate and so they  don't 'need' to activate the checkpoint to prevent mitosis. Let me think about this a bit more and the best way to describe it. BTW In an earlier commentl when I said post replication I meant post start.

**Comment by ValWood** [comment]
I think for  this ticket quite a lot of thinking will be required ;)

Keep thinking, but we will follow up on this when we are all in the same place (maybe during the GO meeting, or if we do not have time, at the following curator meeting).

In the meantime, we can be thinking about  this  more when we annotate to "elongated" flavour terms, and if we spot things annotated which should clearly  be
FYPO:0003481 - viable elongated vegetative cell, elongated upon mitotic entry
instead of 
plain "viable elongated vegetative cell"

we can begin to move them the annotations down, to make less work later....
